redaxo/src/addons/install/lib/api_package_upload.php

```diff
@@ -37,6 +37,12 @@ public function execute()
                 if ($upload['ignore_tests']) {
                     $exclude[] = 'tests';
                 }
+                $packageExclude = rex_addon::get($addonkey)->getProperty('exclude');
```

Comment:
Ich finde die Bezeichnung noch zu unpräzise. Wenn man es als unwissender in der package.yml liest, weiß man nicht was excluded wird und wo/wann.

Wenn es technisch so funktioniert (habe es nicht getestet) finde ich es super so ansonsten

Reply:
Lieber sowas wie `installerExclude` oder `excludeFromInstaller`?